suite-native/message-system/src/components/MessageScreen.tsx

```diff
@@ -0,0 +1,110 @@
+import { Linking } from 'react-native';
+
+import { IconName } from '@suite-common/icons/src';
+import { Box, Button, Pictogram, PictogramVariant, VStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Message, Variant } from '@suite-common/suite-types';
+
+type MessageScreenProps = {
+    message: Message;
+    defaultTitle?: string;
+    defaultContent?: string;
+    secondaryButtonLabel?: string;
+    onSecondaryButtonPress?: () => void;
+};
+
+const screenStyle = prepareNativeStyle(utils => ({
+    backgroundColor: utils.colors.backgroundSurfaceElevation0,
+    position: 'absolute',
+    justifyContent: 'center',
+    alignItems: 'center',
+    left: 0,
+    right: 0,
+    bottom: 0,
+    top: 0,
+    paddingTop: utils.spacings.extraLarge,
+    paddingHorizontal: utils.spacings.large,
+    marginBottom: utils.spacings.extraLarge,
+    flex: 1,
+}));
+
+const buttonsWrapperStyle = prepareNativeStyle(utils => ({
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    bottom: 0,
+    paddingHorizontal: utils.spacings.large,
+    marginBottom: utils.spacings.extraLarge,
+}));
+
+const variantMap = {
+    info: 'green',
+    warning: 'yellow',
+    critical: 'red',
+} as const satisfies Record<Variant, PictogramVariant>;
+
+const iconVariantMap = {
+    info: 'info',
+    warning: 'warningTriangleLight',
+    critical: 'warningTriangleLight',
+} as const satisfies Record<Variant, IconName>;
+
+export const MessageScreen = ({
+    message,
```

Comment:
Same here please

Reply:
I used the same way as MessageBanner did before, but understand. Will update both